media-libs/tg_owt/metadata.xml

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
+	<maintainer type="person">
+		<email>esteve.varela@gmail.com</email>
+		<name>Esteve Varela Colominas</name>
+	</maintainer>
```

Comment:
you cannot be only maintainer listed. you need proxy-maint added as well.
also you need `proxied="yes""` attribute for yourself.

look at `app-arch/lzma/metadata.xml` for an example how to do it.